src/pages/Detail/Detail.js

```diff
@@ -1,5 +1,200 @@
+import styled from 'styled-components';
+import Alert from './Alert';
+import AsideMenu from './AsideMenu/AsideMenu';
+import BookInfoTap from './BookInfoTap/BookInfoTap';
+import BookSummary from './BookSummary/BookSummary';
+import Comments from './Comments/Comments';
+import DeleteModal from '../../components/Modal/DeleteModal';
+import Potal from '../../components/Modal/Potal';
+import ModalLayout from '../../components/Modal/ModalLayout';
 
-const Detail = props => <div></div>;
+const Detail = props => {
+  const [book, setBook] = useState({});
+  const [comments, setComments] = useState([]);
+  const [isOpen, setIsOpen] = useState([false, false, false]);
+  const [value, setValue] = useState('');
+  const [onAlert, setOnAlert] = useState('none');
+  const [openModal, setOpenModal] = useState({
+    type: 'none',
+    id: 0,
+  });
+
+  useEffect(() => {
+    fetch('http://54.180.126.99:8000/books/1')
+      .then(res => res.json())
+      .then(res => setBook(res.RESULT))
+      .catch(err => alert(err));
+  }, []);
+
+  useEffect(() => {
+    getComment();
+  }, []);
+
+  const getComment = () => {
+    fetch('http://54.180.126.99:8000/books/1/comments?id=1')
+      .then(res => res.json())
+      .then(res => setComments(res.comments))
+      .catch(err => alert(err));
+  };
+
+  // 서버 없는 경우 mock data
+  // useEffect(() => {
+  //   fetch('/data/detail.json')
+  //     .then(res => res.json())
+  //     .then(res => setBook(res.results));
+  // }, []);
+
+  // useEffect(() => {
+  //   fetch('/data/detail.json')
+  //     .then(res => res.json())
+  //     .then(res => setComments(res.comments));
+  // }, []);
+
+  const handleInfo = index => {
+    const newIsOpen = isOpen.map((item, idx) => (idx === index ? !item : item));
+    setIsOpen(newIsOpen);
+  };
+
+  const handleLike = id => {
+    fetch(
+      `http://54.180.126.99:8000/books/comments-like?comment_id=${id}&id=1`,
+      {
+        method: 'POST',
+      }
+    )
+      .then(res => res.json())
+      .then(res => res.message === 'SUCCESS' && getComment());
+  };
+
+  const handleAlert = content => {
+    setOnAlert(content);
+    setTimeout(() => {
+      setOnAlert('none');
+    }, 1000);
+  };
+
+  const handleCancle = target => {
+    if (target === 'delete') {
+      setOpenModal({
+        state: 'none',
+        id: 0,
+      });
+    }
+  };
+
+  const handleValue = e => {
+    setValue(e.target.value);
+  };
+
+  const addComment = (e, value) => {
+    fetch('http://54.180.126.99:8000/books/1/comments?id=1', {
+      method: 'POST',
+      body: JSON.stringify({
+        text: value,
+      }),
+    })
+      .then(res => res.json())
+      .then(res => res.message === 'SUCCESS' && getComment());
+
+    setValue('');
+  };
+
+  const handleDeleteButton = id => {
+    setOpenModal({
+      type: 'delete',
+      id,
+    });
+  };
+
+  const deleteComment = id => {
+    fetch(`http://54.180.126.99:8000/books/${id}/comment-delete`, {
```

Comment:
```suggestion
    fetch(`${BASE_URL}/books/${id}/comment-delete`, {
```
ip 주소 부분은 src/config 등의 경로로 별도로 분리해주세요! 유사한 부분도 함께 정리 부탁드립니다

Reply:
넵! 팀원들과 이야기해서 정리할 예정입니다!